kernel/src/hil/i2c.rs

```diff
@@ -177,7 +177,7 @@ pub trait I2CSlave {
 /// Master and Slave modes.
 pub trait I2CMasterSlave: I2CMaster + I2CSlave {}
 // Provide blanket implementations for trait group
-impl<T: I2CMaster + I2CSlave> I2CMasterSlave for T {}
+// impl<T: I2CMaster + I2CSlave> I2CMasterSlave for T {}
```

Comment:
Why is this commented now?

Reply:
I commented this because it conflicts the implementation from `chips/nrf52/src/i2c.rs` at line 364.